ImperativeNavigation/Modules/Coordinator.swift

```diff
@@ -8,6 +8,7 @@ protocol Coordinator {
     func navigateToC()
     func dismissCThenPresentD()
     func navigateToRoot()
+    func popLast()
```

Comment:
```suggestion
    func pop()
```
I would suggest renaming the method to `pop()`. This matches the push-pop convention and removes the redundant `last` keyword.